ui/src/apps/clusterInfo/components/ClusterInfoTable.js

```diff
@@ -99,4 +130,8 @@ function wrapNode(node, comp, id) {
   }
 }
 
+async function deleteTiDBTopology(node, cluster, setCluster) {
+  await client.dashboard.topologyTidbAddressDelete(node.address)
```

Comment:
You need to do something. e.g. Confirm before hide, letting user know what will happen if xxx, and update the list after hide is succeeded.